pyproject.toml

```
@@ -45,6 +45,12 @@
     name = "Contributor-facing changes"
     showcontent = true
 
+[build-system]
+requires = [
+  "setuptools>=45",
+  "wheel",
+  "setuptools_scm[toml]>=6.2"
```

In fact, it's probably nicer to also make this sparse.
```suggestion
  "setuptools >= 45",
  "setuptools_scm[toml] >= 6.2"
```